image-create

```diff
@@ -56,10 +56,12 @@ if args.image in ["fedora-coreos", "services", "selenium", "openshift"]:
 class MachineBuilder:
     def __init__(self, machine):
         self.machine = machine
+        self.iso = "boot" in self.machine.image
```

Comment:
That heuristic is a bit uninutitive, but let's start with that, we can refine it later if we need to. But perhaps let's do `.endswith("-boot")`, just in case.